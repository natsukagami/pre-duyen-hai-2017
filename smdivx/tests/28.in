30 703600310159434
