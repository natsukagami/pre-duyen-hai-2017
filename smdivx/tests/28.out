100000000000000121187736178152
